resources/lang/en/absent.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+return [
+    'success' => 'You send request successful!',
+    'fail' => 'You absent requested one time today!',
```

Comment:
key should be more specific, `fail` is too common for the text `You absent requested one time today`